src/Apps/FeatureAKG/Components/Feature.tsx

```diff
@@ -173,24 +202,38 @@ const Section: React.FC<SectionProps> = props => {
   )
 }
 
-const VideoSection: React.FC<{ src: string }> = props => {
-  // Constrain the height, add 1px for the border
+const Video: React.FC<{ src: string }> = props => {
   return (
-    <Box mb="-40px" height="591px">
-      <SectionSeparator mt="4" />
-      <Box textAlign="center">
-        <Video src={props.src} />
-      </Box>
-    </Box>
+    <StyledVideo
+      src={props.src}
+      autoPlay
+      loop
+      muted
+      playsInline
+      controls={false}
+    />
   )
 }
 
-const Video: React.FC<{ src: string }> = props => {
+const StyledVideo = styled("video")`
+  width: 100%;
+  height: auto;
+`
+
+const ImageSection: React.FC<{ src: string; aspectRatio: number }> = props => {
   return (
-    <video autoPlay loop muted playsInline controls={false} src={props.src} />
+    <BorderedSection>
+      <ResponsiveImage lazyLoad src={props.src} ratio={props.aspectRatio} />
```

Comment:
This is where we handle the gif-- lazy loading works but we're still getting the accordion effect. I'll continue looking but it can be a follow up.